JSDemos/Demos/Calendar/Overview/Angular/description.md

```diff
@@ -18,9 +18,9 @@ Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration
 
 If you need to disable specific dates, use the [disabledDates](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) property. Toggle the "Disable weekend" checkbox to see how this setting affects the component's behavior. You can specify either an array of predefined dates or a function that determines whether a date is available.
 
-## Set the Beginning of the Week
+## Specify First Day of Week and Display Week Numbers
 
-To specify the first day of the week, pass its index (0 - for Sunday, 1 - for Monday, and so on) to the [firstDayOfWeek](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#firstDayOfWeek) property.
+To specify the first day of the week, assign its index (0 - for Sunday, 1 - for Monday, and so on) to the [firstDayOfWeek](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#firstDayOfWeek) property. You can also display a column with week numbers. For this, enable the [showWeekNumbers](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#showWeekNumbers) property.
 
 ## Handle Value Change
 
```

Comment:
Please make the following change to the line below.

Line 27:
Please remove the extra space at the end of the first sentence (before the period).